apps/web/playwright/lib/teardown.ts

```diff
@@ -54,9 +54,18 @@ export const deleteAllPaymentsByEmail = async (email: string) => {
 };
 
 export const deleteAllPaymentCredentialsByEmail = async (email: string) => {
-  await prisma.user.update({
+  const user = await prisma.user.findFirst({
     where: {
       email,
+      linkedBy: null,
+    },
+    select: {
+      id: true,
+    },
+  });
+  await prisma.user.update({
+    where: {
+      id: user?.id,
```

Comment:
This tackles payment-related metadata for regular users, when org related e2e tests require payment-related management from the test, new function will be added.